src/UpgradeableProxy.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { IUpgradeableProxy } from "./interfaces/IUpgradeableProxy.sol";
+
+contract UpgradeableProxy is IUpgradeableProxy {
+
+    address public override implementation;
+
+    mapping (address => uint256) public override wards;
+
+    modifier auth {
+        require(wards[msg.sender] == 1, "UpgradeableProxy/not-authed");
```

Comment:
Should be `not-authorized` instead of `not-authed`. This was a mistake in the cropper repo.

Reply:
Updated